config/application.rb

```diff
@@ -1,6 +1,12 @@
 require File.expand_path('../boot', __FILE__)
 require File.expand_path('../preinitializer', __FILE__)
-require 'rails/all'
+require 'rails'
+require 'active_record/railtie'
+require 'action_controller/railtie'
+require 'action_view/railtie'
+require 'action_mailer/railtie'
+require 'active_job/railtie'
+require 'sprockets/railtie'
```

Comment:
@chrisarcand What does this do?  What are preventing from loading?  I can't recall why we did rails/all but it's probably really old, maybe even from rails 3.x days.